subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r22/ClientShutdownCrossVersionSpec.groovy

```diff
@@ -41,12 +39,13 @@ class ClientShutdownCrossVersionSpec extends ToolingApiSpecification {
 
     def "can shutdown tooling API session when no operations have been executed"() {
         given:
-        toolingApi.close();
+        toolingApi.close()
 
         when:
         toolingApi.withConnection {}
         then:
         IllegalStateException e = thrown()
+        e
```

Comment:
I was trying to clean up a warning... not sure if the line, as it is, forces a cast to the exception to verify it's working or not... but I'll just revert it for now.